src/drv_oculus_rift/rift.h

```diff
@@ -1,5 +1,6 @@
 // Copyright 2013, Fredrik Hultin.
 // Copyright 2013, Jakob Bornecrantz.
+// Copyright 2015-2016 Philipp Zabel
```

Comment:
For the creative achievement of naming variables? Fair enough, I suppose. Given the amount of code written anew and/or lifted from ouvrt, I'd say you could add both of us to the list in rift.c as well.

Reply:
Will do